apisix/plugins/prometheus/exporter.lua

```diff
@@ -122,6 +123,10 @@ function _M.http_init(prometheus_enabled_in_stream)
             "Etcd modify index for APISIX keys",
             {"key"})
 
+    metrics.shared_dict = prometheus:gauge("shared_dict",
+            "nginx shared DICT of APISIX",
+            {"key"})
```

Comment:
This is not a standard way to name a prometheus metric.
We should distinguish them with name, instead of abusing the label.

They should be:
```
apisix_shared_dict_free_space{name="internal-status"}
...
```
and the unit should be byte.

See https://prometheus.io/docs/practices/naming/#metric-names

Reply:
> 

Modify the initialization with
```lua
    metrics.shared_dict_capacity_bytes = prometheus:gauge("shared_dict_capacity_bytes",
            "nginx shared DICT capacity of APISIX",
            {"name"})

    metrics.shared_dict_free_space_bytes = prometheus:gauge("shared_dict_free_space_bytes",
            "nginx shared DICT free_space of APISIX",
            {"name"})
```
Is my understanding correct?

Reply:
Yes. The description can be more clear, like `The total number of client requests since APISIX started`